js/card.js

```diff
@@ -0,0 +1,74 @@
+import {getNoun} from './util.js';
+
+const card = document.querySelector ('#card').content.querySelector('.popup');
+
+const typeHouseDict = {
+  palace: 'Дворец',
+  flat: 'Квартира',
+  house: 'Дом',
+  bungalow: 'Бунгало',
+  hotel: 'Отель'
+};
+
+const hidesBlock = (block, content) => {
+
+  if (!content || content.length === 0) {
+    block.style.display = 'none';
+
+  }
+
+};
```

Comment:
Лишние отступы, если в коде if один отделять его пробелами не нужно
```suggestion
  if (!content || content.length === 0) {
    block.style.display = 'none';
  }
};
```